src/Sarif.FunctionalTests/ConverterTestData/PREfast/AA_IndexAlias.cpp.xml.sarif

```diff
@@ -43,10 +43,10 @@
                     "startLine":5
                   }
                 },
-                "id":"1",
+                "step":0,
```

Comment:
do we want the converted code to explicitly include the 'id' from the PREfast log? what about the native PREfast 'importance' property (which consists of either 'important' or 'essential, i think). we can include this data for completeness. arguably, since we can directly convert to an existing SARIF concept with no loss of fidelity it can be omitted (but you need to know the rules of conversion). thoughts?